### PR TITLE
feat: [M1749] benthic photo quadrat implementation in dashboard

### PR DESCRIPTION
**Pull request by tienhoah**

### CODE REVIEW

**Comment by tienhoah** [comment]
[benthic photo quadrat implementation in dashboard](https://trello.com/c/vN5nttE5/1749-benthic-photo-quadrat-implementation-in-dashboard)

File: src/components/InformationCard.js [hunk 1]
```diff
@@ -38,16 +37,15 @@ const cardStyle = makeStyles(theme => ({
 }));
 
 const InformationCard = ({
+  bleachingProtocolSubItems,
```
Comment (mmnoo) [inline src/components/InformationCard.js:40]: nice alphabetization

File: src/components/PieChart.js [hunk 2]
```diff
@@ -175,7 +177,7 @@ const PieChart = ({ protocolName, chartContent, setToPrivate, privateLabel }) =>
 
 PieChart.propTypes = {
   chartContent: PropTypes.array,
-  setToPrivate: PropTypes.bool,
+  isPrivatePolicy: PropTypes.bool,
```
Comment (mmnoo) [inline src/components/PieChart.js:180]: nice one on naming.

File: src/components/SiteDetail.js [hunk 3]
```diff
@@ -72,63 +73,46 @@ const SiteDetail = ({ selectSite, projectFishFamilies }) => {
...
-      `This data is unavailable because ${protocolTitle} Sample Units are set to Private for this project.`;
-
-    const convertContent = content => {
-      let contentArr = [];
-      for (const item in content) {
-        const newObject = {};
-        newObject[item] = content[item];
-        contentArr.push(newObject);
+  const siteChartCards = useMemo(() => {
+    const selectedSiteProtocols = currentSelectedSite?.protocols;
```
Comment (mmnoo) [inline src/components/SiteDetail.js:77]: is selectedSiteProtocols an array? or object? Is this a weird api thing?

**Review by mmnoo — APPROVED** [review]

File: src/components/SiteDetail.js (hunk 3 above)
Comment (tienhoah) [reply src/components/SiteDetail.js:77]: its an object

**Pull request merged**